src/miral/application_selector.cpp

```diff
@@ -19,6 +19,7 @@
 #include <miral/application.h>
 #include <mir/log.h>
 #include <mir/scene/session.h>
+#include <unordered_set>
```

Comment:
Why?

Reply:
Left over from when I tried to "optimize" the code. 2 minutes in and I realized it's written like that for a reason